benchexec/runexecutor.py

```diff
@@ -648,6 +650,8 @@ def execute_run(
         @param files_size_limit: None or maximum size of files that may be written.
         @param error_filename: the file where the error output should be written to (default: same as output_filename)
         @param write_headers: Write informational headers to the output and the error file if separate (default: True)
+        @param parent_setup_fn: A function that gets called in the parent process before the tool is allowed to run (default: noop fn). When used with use_namespaces=True, the function recieves child_pid and grandchild_pid.
+        @param parent_cleanup_fn: A function that gets called after the tool finishes running with the return value of parent_setup_fn.
```

Comment:
We should think about how to make it easy to ensure compatibility in the future here. For example, what can we do such that we can add more information in the future?